docs/vgclog_examples.md

```diff
@@ -0,0 +1,595 @@
+|concurrent mark      |intermediate |concurrent                     | none|`<concurrent-trace-info>` records the progress of the concurrent mark increment|
+|final collection     |final        | STW              | `<concurrent-global-final>`  |The increment is typically triggered when a card cleaning threshold is reached. The completion of a tracing phase can also trigger the increment. Operations include a final concurrent mark, a sweep, and an optional class unload and compact.|
+
+The global GC cycle follows a general structure in the verbose GC log as shown. The lines are indented to help illustrate the flow and attributes and some child elements are omitted for clarity. Multiple partial GC cycles can start and finish between the start and end of a global GC cycle. In this example, the structure includes a single partial GC cycle within the global cycle:
+
+```xml
+
+<concurrent-kickoff/>       (global cycle 1st increment recorded)
+
+<exclusive-start/>          (STW pause starts)
+
+<cycle-start/>              (global cycle starts)
+
+<exclusive-end/>            (STW pause ends)
+
+(mutator threads running, global cycle concurrent increment running concurrently)
+
+
+<exclusive-start/>          (STW for partial GC cycle starts)
+
+
+...                         (partial GC cycle starts and completes)
+
+
+<exclusive-end/>            (STW for partial GC cycle ends)
+
+(mutator threads running, global cycle concurrent increment running concurrently)
+
+<exclusive-start/>          (STW pause starts)
+
+<concurrent-global-final/>  (global cycle final increment recorded)
+
+<gc-start/>                 (global cycle final increment starts)
+
+<allocation-stats/>         (snapshot of application threads status taken before cycle started)
+
+<mem-info>                  (memory status before operations)
+
+<mem></mem>                 (status of different types of memory)
+
+</mem-info>
+
+</gc-start>
+
+<gc-op> “type=rs-scan"</gc-op>        (remembered set scan completed)
+
+<gc-op>”type=card-cleaning" </gc-op>  (card cleaning completed)
+
+<gc-op> “type=mark”</gc-op>           (final mark operation and weak roots processing completed)
+
+<gc-op> “type=classunload”</gc-op>    (class unload operation completed)
+
+<gc-op ”type=sweep” />                (sweep operation completed)
+
+<gc-end>                     (global cycle final increment ends)
+
+<mem-info>                   (memory status after operations)
+
+<mem></mem>                  (status of different types of memory)
+
+</mem-info>
+
+</gc-end> 
+
+</cycle-end>                 (global cycle ends)
+
+<exclusive-end>              (STW pause ends)
+
+<exclusive-start>            (STW pause starts)
+...
+
+```
+
+
+The first activity in the cycle is recorded by a `<concurrent-kickoff>` element, which records the start of the first of three increments that make up a `gencon` Global GC cycle. The `<concurrent-kickoff>` element records:
```

Comment:
Fragment with colon. Suggest "The `<concurrent-kickoff>` element records the following information:"